x-pack/plugins/observability/public/pages/overview/data_sections.tsx

```diff
@@ -11,24 +11,21 @@ import { MetricsSection } from '../../components/app/section/metrics';
 import { APMSection } from '../../components/app/section/apm';
 import { UptimeSection } from '../../components/app/section/uptime';
 import { UXSection } from '../../components/app/section/ux';
-import {
-  HasDataResponse,
-  ObservabilityFetchDataPlugins,
-  UXHasDataResponse,
-} from '../../typings/fetch_overview_data';
+import { UXHasDataResponse } from '../../typings/fetch_overview_data';
+import { HasDataMap } from '../../context/has_data_context';
 
 interface Props {
   bucketSize: string;
   absoluteTime: { start?: number; end?: number };
   relativeTime: { start: string; end: string };
-  hasData: Record<ObservabilityFetchDataPlugins, HasDataResponse>;
+  hasData?: Partial<HasDataMap>;
 }
 
 export function DataSections({ bucketSize, hasData, absoluteTime, relativeTime }: Props) {
   return (
     <EuiFlexItem grow={false}>
       <EuiFlexGroup direction="column">
-        {hasData?.infra_logs && (
+        {hasData?.infra_logs?.hasData && (
```

Comment:
We can remove these statements, and add a `hidden` prop on the EUIFlexItem component. That will ensure the component itself will start loading the data.

Reply:
Are you sure the hidden property work? I haven't found any description of it here https://elastic.github.io/eui/#/layout/flex, and I tried it and it always shows the child element.

Reply:
It's a native HTML attribute. It should work, but if it doesn't, there might be some styles from EUI overriding it, or React special-casing it. We don't have to use `hidden`, but the point is to render the component so it starts the request, but hide it from the UI.